src/UI/MvePlayer.cpp

```diff
@@ -837,8 +819,10 @@ void MvePlayer::_processChunk()
         return;
     }
 
-    for (auto opcode : *_chunk->opcodes())
+    auto opcodes = _chunk->opcodes();
+    for (unsigned int i=0; i < opcodes->size();i++)
```

Comment:
Can we use range-based for loop here? Like 

`for (const auto& opcode : opcodes) ...`